corehq/apps/translations/models.py

```diff
@@ -92,3 +100,45 @@ def get_translations(cls, lang, key=None, one=False):
                 return dict([(key, val[0]) for key, val in translations.items()])
             else:
                 return translations
+
+
+FIELD_NAME_HELP = """
+Usually the string in either case list or detail under 'property'.
+This could be an xpath or case property name.
+If it is an ID Mapping then the property should be '<property> (ID Mapping Text)'.
+For the values each value should be '<id mapping value> (ID Mapping Value)'.
+Example: case detail for tasks_type would have entries:
+    tasks_type (ID Mapping Text)
+    child (ID Mapping Value)
+    pregnancy (ID Mapping Value)
```

Comment:
nit: can we just say that this should match what the user see under `property` column in bulk translations download so that they can just pick it from there instead of trying to figure out what to put ?

Reply:
It does seem useful to have that as an example. I added a note about it being the same as in bulk translations above